src/features/rewards/grantClaim/index.tsx

```diff
@@ -10,18 +10,18 @@ const gift = require('./assets/gift')
 
 export interface Props {
   id?: string
-  onClick: () => void
+  onClaim: () => void
```

Comment:
Should this be `onClaim?:` ?

Reply:
can you elaborate more why this action should be optional

Reply:
my mistake, i was thinking actions would be kept optional for now but that doesn't make sense with this one

Reply:
actions are only optional when this action can't be preformed because of reason X. For example we have optional action for opening view monthly report, because we are not using this action in one other scenario